BaragonService/src/main/java/com/hubspot/baragon/service/BaragonServiceModule.java

```diff
@@ -265,17 +281,18 @@ public String providesLocalHostnameProperty(BaragonConfiguration config) {
 
       return addr.getHostName();
     } catch (UnknownHostException e) {
-      throw new RuntimeException("No local hostname found, unable to start without functioning local networking (or configured hostname)", e);
+      throw new RuntimeException(
+          "No local hostname found, unable to start without functioning local networking (or configured hostname)", e);
     }
   }
 
   @Provides
   @Singleton
   @Named(BaragonDataModule.BARAGON_SERVICE_LEADER_LATCH)
   public LeaderLatch providesServiceLeaderLatch(BaragonConfiguration config,
-                                                BaragonWorkerDatastore datastore,
```

Comment:
This is a super nit but why did all this spacing change?

Reply:
Oh https://github.com/HubSpot/Baragon/pull/370/files?w=1 solves that.

Reply:
Agreed, I've looked into this a few times many months ago and gave up trying to fix prettier here. Agree would be a nice to have, can take a quick look again maybe with a new set of eyes what we can do for formatting here, but there's something different in this repo vs our normal formatting conventions